searx/__init__.py

```diff
@@ -30,6 +30,7 @@
 
 searx_dir = abspath(dirname(__file__))
 engine_dir = dirname(realpath(__file__))
+static_path = abspath(join(dirname(__file__), 'static'))
```

Comment:
Missed support of  [custom static path](https://github.com/asciimoo/searx/blob/master/searx/settings.yml#L21)